packages/e2e-tests/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@sentry-internal/e2e-tests",
+  "version": "0.0.0",
+  "license": "MIT",
```

Comment:
l: Main repo license is BSD-3, so if we write MIT here we’ll need to copy in the MIT license iirc

https://github.com/getsentry/sentry-javascript/blob/master/LICENSE

Reply:
👍 0709dd66208f801a4c18e162b1c1e7f1ab4aa325